html/en/beats/winlogbeat/master/_winlogbeat_overview.html
html/en/beats/winlogbeat/master/add-cloud-metadata.html
html/en/beats/winlogbeat/master/add-docker-metadata.html
html/en/beats/winlogbeat/master/add-fields.html
html/en/beats/winlogbeat/master/add-host-metadata.html
html/en/beats/winlogbeat/master/add-kubernetes-metadata.html
html/en/beats/winlogbeat/master/add-labels.html
html/en/beats/winlogbeat/master/add-locale.html
html/en/beats/winlogbeat/master/add-observer-metadata.html
html/en/beats/winlogbeat/master/add-process-metadata.html
html/en/beats/winlogbeat/master/add-tags.html
html/en/beats/winlogbeat/master/beats-basic-auth.html
html/en/beats/winlogbeat/master/beats-system-user.html
html/en/beats/winlogbeat/master/beats-tls.html
html/en/beats/winlogbeat/master/bogus-computer-name-fields.html
html/en/beats/winlogbeat/master/command-line-options.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/command-line-options.html
@@ -477,37 +477,41 @@
               </code>
               .
              </dd>
-             <dt>
-              <span class="term">
-               <span class="strong strong">
-                <strong>
-                 <code class="literal">
-                  ilm-policy
-                 </code>
-                </strong>
-               </span>
-              </span>
-             </dt>
-             <dd>
-              Exports the index lifecycle management policy to stdout. You can specify the
-              <code class="literal">
-               --es.version
-              </code>
-              and a
-              <code class="literal">
-               --dir
-              </code>
-              to which the policy should be exported as a file rather than exporting to
-              <code class="literal">
-               stdout
-              </code>
-              .
-             </dd>
             </dl>
            </div>
-           <p>
+           <div class="variablelist">
             <a id="ilm-policy-subcommand">
             </a>
+            <dl class="variablelist">
+             <dt>
+              <span class="term">
+               <span class="strong strong">
+                <strong>
+                 <code class="literal">
+                  ilm-policy
+                 </code>
+                </strong>
+               </span>
+              </span>
+             </dt>
+             <dd>
+              Exports the index lifecycle management policy to stdout. You can specify the
+              <code class="literal">
+               --es.version
+              </code>
+              and a
+              <code class="literal">
+               --dir
+              </code>
+              to which the policy should be exported as a file rather than exporting to
+              <code class="literal">
+               stdout
+              </code>
+              .
+             </dd>
+            </dl>
+           </div>
+           <p>
             <span class="strong strong">
              <strong>
               FLAGS
html/en/beats/winlogbeat/master/community-id.html
html/en/beats/winlogbeat/master/config-winlogbeat-logstash.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/config-winlogbeat-logstash.html
@@ -171,6 +171,9 @@
             <img alt="Important" src="images/icons/important.png"/>
            </div>
            <div class="admon_content">
+            <h3>
+             Prerequisite
+            </h3>
             <p>
              To send events to Logstash, you also need to create a Logstash configuration pipeline that listens for incoming Beats connections and indexes the received events into Elasticsearch. For more information, see the section about
              <a class="ulink" href="https://www.elastic.co/guide/en/elastic-stack-get-started/master/get-started-elastic-stack.html#logstash-setup" target="_top">
html/en/beats/winlogbeat/master/configuration-dashboards.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-dashboards.html
@@ -213,7 +213,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_enabled_literal">
+           <a id="_setup_dashboards_enabled">
            </a>
            <code class="literal">
             setup.dashboards.enabled
@@ -308,7 +308,7 @@
            The file archive (zip file) that contains the dashboards to load. If this option is set, Winlogbeat looks for a dashboard archive in the specified path instead of using the local directory.
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_beat_literal">
+           <a id="_setup_dashboards_beat">
            </a>
            <code class="literal">
             setup.dashboards.beat
@@ -325,7 +325,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_kibana_index_literal">
+           <a id="_setup_dashboards_kibana_index">
            </a>
            <code class="literal">
             setup.dashboards.kibana_index
@@ -341,7 +341,7 @@
            </code>
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_index_literal">
+           <a id="_setup_dashboards_index">
            </a>
            <code class="literal">
             setup.dashboards.index
@@ -367,7 +367,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_setup_dashboards_always_kibana_literal">
+           <a id="_setup_dashboards_always_kibana">
            </a>
            <code class="literal">
             setup.dashboards.always_kibana
@@ -384,7 +384,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_retry_enabled_literal">
+           <a id="_setup_dashboards_retry_enabled">
            </a>
            <code class="literal">
             setup.dashboards.retry.enabled
@@ -397,7 +397,7 @@
            If this option is set to true, and Kibana is not reachable at the time when dashboards are loaded, Winlogbeat will retry to reconnect to Kibana instead of exiting with an error. Disabled by default.
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_retry_interval_literal">
+           <a id="_setup_dashboards_retry_interval">
            </a>
            <code class="literal">
             setup.dashboards.retry.interval
@@ -410,7 +410,7 @@
            Duration interval between Kibana connection retries. Defaults to 1 second.
           </p>
           <h4>
-           <a id="_literal_setup_dashboards_retry_maximum_literal">
+           <a id="_setup_dashboards_retry_maximum">
            </a>
            <code class="literal">
             setup.dashboards.retry.maximum
html/en/beats/winlogbeat/master/configuration-general-options.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-general-options.html
@@ -192,7 +192,7 @@
 tags: ["service-X", "web-tier"]</pre>
           </div>
           <h4>
-           <a id="_literal_name_literal">
+           <a id="_name">
            </a>
            <code class="literal">
             name
@@ -219,7 +219,7 @@
            <pre class="programlisting prettyprint lang-yaml">name: "my-shipper"</pre>
           </div>
           <h4>
-           <a id="_literal_tags_literal">
+           <a id="_tags">
            </a>
            <code class="literal">
             tags
@@ -269,7 +269,7 @@
            <pre class="programlisting prettyprint lang-yaml">fields: {project: "myproject", instance-id: "574734885120952459"}</pre>
           </div>
           <h4>
-           <a id="_literal_fields_under_root_literal">
+           <a id="_fields_under_root">
            </a>
            <code class="literal">
             fields_under_root
@@ -299,7 +299,7 @@
   region: us-east-1</pre>
           </div>
           <h4>
-           <a id="_literal_processors_literal">
+           <a id="_processors">
            </a>
            <code class="literal">
             processors
@@ -321,7 +321,7 @@
            for information about specifying processors in your config.
           </p>
           <h4>
-           <a id="_literal_max_procs_literal">
+           <a id="_max_procs">
            </a>
            <code class="literal">
             max_procs
html/en/beats/winlogbeat/master/configuration-logging.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-logging.html
@@ -220,7 +220,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_logging_to_stderr_literal">
+           <a id="_logging_to_stderr">
            </a>
            <code class="literal">
             logging.to_stderr
@@ -237,7 +237,7 @@
            command line option.
           </p>
           <h4>
-           <a id="_literal_logging_to_syslog_literal">
+           <a id="_logging_to_syslog">
            </a>
            <code class="literal">
             logging.to_syslog
@@ -260,7 +260,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_logging_to_eventlog_literal">
+           <a id="_logging_to_eventlog">
            </a>
            <code class="literal">
             logging.to_eventlog
@@ -273,7 +273,7 @@
            When true, writes all logging output to the Windows Event Log.
           </p>
           <h4>
-           <a id="_literal_logging_to_files_literal">
+           <a id="_logging_to_files">
            </a>
            <code class="literal">
             logging.to_files
@@ -426,7 +426,7 @@
            also sets the debug log level).
           </p>
           <h4>
-           <a id="_literal_logging_metrics_enabled_literal">
+           <a id="_logging_metrics_enabled">
            </a>
            <code class="literal">
             logging.metrics.enabled
@@ -448,7 +448,7 @@
            Note that we currently offer no backwards compatible guarantees for the internal metrics and for this reason they are also not documented.
           </p>
           <h4>
-           <a id="_literal_logging_metrics_period_literal">
+           <a id="_logging_metrics_period">
            </a>
            <code class="literal">
             logging.metrics.period
@@ -461,7 +461,7 @@
            The period after which to log the internal metrics. The default is 30s.
           </p>
           <h4>
-           <a id="_literal_logging_files_path_literal">
+           <a id="_logging_files_path">
            </a>
            <code class="literal">
             logging.files.path
@@ -478,7 +478,7 @@
            section for details.
           </p>
           <h4>
-           <a id="_literal_logging_files_name_literal">
+           <a id="_logging_files_name">
            </a>
            <code class="literal">
             logging.files.name
@@ -497,7 +497,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_logging_files_rotateeverybytes_literal">
+           <a id="_logging_files_rotateeverybytes">
            </a>
            <code class="literal">
             logging.files.rotateeverybytes
@@ -510,7 +510,7 @@
            The maximum size of a log file. If the limit is reached, a new log file is generated. The default size limit is 10485760 (10 MB).
           </p>
           <h4>
-           <a id="_literal_logging_files_keepfiles_literal">
+           <a id="_logging_files_keepfiles">
            </a>
            <code class="literal">
             logging.files.keepfiles
@@ -527,7 +527,7 @@
            options has to be in the range of 2 to 1024 files.
           </p>
           <h4>
-           <a id="_literal_logging_files_permissions_literal">
+           <a id="_logging_files_permissions">
            </a>
            <code class="literal">
             logging.files.permissions
@@ -566,7 +566,7 @@
            </ul>
           </div>
           <h4>
-           <a id="_literal_logging_files_interval_literal">
+           <a id="_logging_files_interval">
            </a>
            <code class="literal">
             logging.files.interval
@@ -579,7 +579,7 @@
            Enable log file rotation on time intervals in addition to size-based rotation. Intervals must be at least 1s. Values of 1m, 1h, 24h, 7*24h, 30*24h, and 365*24h are boundary-aligned with minutes, hours, days, weeks, months, and years as reported by the local system clock. All other intervals are calculated from the unix epoch. Defaults to disabled.
           </p>
           <h4>
-           <a id="_literal_logging_files_rotateonstartup_literal">
+           <a id="_logging_files_rotateonstartup">
            </a>
            <code class="literal">
             logging.files.rotateonstartup
@@ -592,7 +592,7 @@
            If the log file already exists on startup, immediately rotate it and start writing to a new file instead of appending to the existing one. Defaults to true.
           </p>
           <h4>
-           <a id="_literal_logging_json_literal">
+           <a id="_logging_json">
            </a>
            <code class="literal">
             logging.json
@@ -605,7 +605,7 @@
            When true, logs messages in JSON format. The default is false.
           </p>
           <h4>
-           <a id="_literal_logging_files_redirect_stderr_literal_phrase_role_experimental_phrase">
+           <a id="_logging_files_redirect_stderr">
            </a>
            <code class="literal">
             logging.files.redirect_stderr
@@ -672,17 +672,17 @@
           </p>
           <p>
            <code class="literal">
-            2017-12-17T18:54:16.241-0500   INFO    logp/core_test.go:13    unnamed global logger
-           </code>
-          </p>
-          <p>
-           <code class="literal">
-            2017-12-17T18:54:16.242-0500   INFO    [example]       logp/core_test.go:16    some message
-           </code>
-          </p>
-          <p>
-           <code class="literal">
-            2017-12-17T18:54:16.242-0500   INFO    [example]       logp/core_test.go:19    some message    {"x": 1}
+            2017-12-17T18:54:16.241-0500	INFO	logp/core_test.go:13	unnamed global logger
+           </code>
+          </p>
+          <p>
+           <code class="literal">
+            2017-12-17T18:54:16.242-0500	INFO	[example]	logp/core_test.go:16	some message
+           </code>
+          </p>
+          <p>
+           <code class="literal">
+            2017-12-17T18:54:16.242-0500	INFO	[example]	logp/core_test.go:19	some message	{"x": 1}
            </code>
           </p>
          </div>
html/en/beats/winlogbeat/master/configuration-monitor.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-monitor.html
@@ -189,7 +189,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_monitoring_enabled_literal">
+               <a id="_monitoring_enabled">
                </a>
                <code class="literal">
                 monitoring.enabled
@@ -225,7 +225,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_monitoring_elasticsearch_literal">
+               <a id="_monitoring_elasticsearch">
                </a>
                <code class="literal">
                 monitoring.elasticsearch
@@ -246,7 +246,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_bulk_max_size_literal_6">
+               <a id="_bulk_max_size_6">
                </a>
                <code class="literal">
                 bulk_max_size
@@ -275,7 +275,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_backoff_init_literal_4">
+               <a id="_backoff_init_4">
                </a>
                <code class="literal">
                 backoff.init
@@ -304,7 +304,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_backoff_max_literal_4">
+               <a id="_backoff_max_4">
                </a>
                <code class="literal">
                 backoff.max
@@ -325,7 +325,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_compression_level_literal_4">
+               <a id="_compression_level_4">
                </a>
                <code class="literal">
                 compression_level
@@ -362,7 +362,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_headers_literal_2">
+               <a id="_headers_2">
                </a>
                <code class="literal">
                 headers
@@ -387,7 +387,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_hosts_literal_3">
+               <a id="_hosts_3">
                </a>
                <code class="literal">
                 hosts
@@ -412,7 +412,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_max_retries_literal_5">
+               <a id="_max_retries_5">
                </a>
                <code class="literal">
                 max_retries
@@ -441,7 +441,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_parameters_literal_2">
+               <a id="_parameters_2">
                </a>
                <code class="literal">
                 parameters
@@ -462,7 +462,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_password_literal_4">
+               <a id="_password_4">
                </a>
                <code class="literal">
                 password
@@ -483,7 +483,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_metrics_period_literal">
+               <a id="_metrics_period">
                </a>
                <code class="literal">
                 metrics.period
@@ -504,7 +504,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_state_period_literal">
+               <a id="_state_period">
                </a>
                <code class="literal">
                 state.period
@@ -525,7 +525,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_protocol_literal">
+               <a id="_protocol">
                </a>
                <code class="literal">
                 protocol
@@ -562,7 +562,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_proxy_url_literal_4">
+               <a id="_proxy_url_4">
                </a>
                <code class="literal">
                 proxy_url
@@ -587,7 +587,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_timeout_literal_5">
+               <a id="_timeout_5">
                </a>
                <code class="literal">
                 timeout
@@ -612,7 +612,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_ssl_literal_5">
+               <a id="_ssl_5">
                </a>
                <code class="literal">
                 ssl
@@ -643,7 +643,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_username_literal_3">
+               <a id="_username_3">
                </a>
                <code class="literal">
                 username
html/en/beats/winlogbeat/master/configuration-output-codec.html
html/en/beats/winlogbeat/master/configuration-path.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-path.html
@@ -216,7 +216,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_home_literal">
+           <a id="_home">
            </a>
            <code class="literal">
             home
@@ -235,7 +235,7 @@
            <pre class="programlisting prettyprint lang-yaml">path.home: /usr/share/beats</pre>
           </div>
           <h4>
-           <a id="_literal_config_literal">
+           <a id="_config">
            </a>
            <code class="literal">
             config
@@ -254,7 +254,7 @@
            <pre class="programlisting prettyprint lang-yaml">path.config: /usr/share/beats/config</pre>
           </div>
           <h4>
-           <a id="_literal_data_literal">
+           <a id="_data">
            </a>
            <code class="literal">
             data
@@ -291,7 +291,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_logs_literal">
+           <a id="_logs">
            </a>
            <code class="literal">
             logs
html/en/beats/winlogbeat/master/configuration-ssl.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-ssl.html
@@ -182,6 +182,7 @@
              <a class="link" href="setup-kibana-endpoint.html" title="Configure the Kibana endpoint">
               Kibana endpoint
              </a>
+             endif::["winlogbeat"!="apm-server"]
             </li>
            </ul>
           </div>
@@ -233,7 +234,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_enabled_literal_7">
+           <a id="_enabled_7">
            </a>
            <code class="literal">
             enabled
@@ -280,7 +281,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_certificate_authorities_literal">
+           <a id="_certificate_authorities">
            </a>
            <code class="literal">
             certificate_authorities
@@ -338,7 +339,7 @@
            is specified.
           </p>
           <h4>
-           <a id="_literal_key_passphrase_literal">
+           <a id="_key_passphrase">
            </a>
            <code class="literal">
             key_passphrase
@@ -355,7 +356,7 @@
            file.
           </p>
           <h4>
-           <a id="_literal_supported_protocols_literal">
+           <a id="_supported_protocols">
            </a>
            <code class="literal">
             supported_protocols
@@ -395,7 +396,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_verification_mode_literal">
+           <a id="_verification_mode">
            </a>
            <code class="literal">
             verification_mode
@@ -431,7 +432,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_cipher_suites_literal">
+           <a id="_cipher_suites">
            </a>
            <code class="literal">
             cipher_suites
@@ -551,7 +552,7 @@
            </ul>
           </div>
           <h4>
-           <a id="_literal_curve_types_literal">
+           <a id="_curve_types">
            </a>
            <code class="literal">
             curve_types
@@ -583,7 +584,7 @@
            </ul>
           </div>
           <h4>
-           <a id="_literal_renegotiation_literal">
+           <a id="_renegotiation">
            </a>
            <code class="literal">
             renegotiation
html/en/beats/winlogbeat/master/configuration-template.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-template.html
@@ -363,7 +363,7 @@
               cross-cluster replication
              </a>
              to replicate Winlogbeat indices to another cluster, you will need to add additional template settings to
-             <a class="ulink" href="https://www.elastic.co/guide/en/elasticsearch/reference/master/ccr-requirements.html#ccr-overview-beats" target="_top">
+             <a class="ulink" href="https://www.elastic.co/guide/en/elastic-stack-overview/master/ccr-requirements.html#ccr-overview-beats" target="_top">
               enable soft deletes
              </a>
              on the underlying indices.
html/en/beats/winlogbeat/master/configuration-winlogbeat-options.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuration-winlogbeat-options.html
@@ -207,7 +207,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_registry_file_literal">
+           <a id="_registry_file">
            </a>
            <code class="literal">
             registry_file
@@ -241,7 +241,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_shutdown_timeout_literal">
+           <a id="_shutdown_timeout">
            </a>
            <code class="literal">
             shutdown_timeout
@@ -287,7 +287,7 @@
            <pre class="programlisting prettyprint lang-yaml">winlogbeat.shutdown_timeout: 30s</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_literal">
+           <a id="_event_logs">
            </a>
            <code class="literal">
             event_logs
@@ -314,7 +314,7 @@
   - name: Application</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_batch_read_size_literal">
+           <a id="_event_logs_batch_read_size">
            </a>
            <code class="literal">
             event_logs.batch_read_size
@@ -428,7 +428,7 @@
   - name: 'C:\backup\sysmon-2019.08.evtx'</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_ignore_older_literal">
+           <a id="_event_logs_ignore_older">
            </a>
            <code class="literal">
             event_logs.ignore_older
@@ -446,7 +446,7 @@
     ignore_older: 168h</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_forwarded_literal">
+           <a id="_event_logs_forwarded">
            </a>
            <code class="literal">
             event_logs.forwarded
@@ -467,7 +467,7 @@
            This settings allows Winlogbeat to optimize reads for forwarded events that are already rendered. When the value is true Winlogbeat does not attempt to render the event using message files from the host computer. The Windows Event Collector subscription should be configured to use the "RenderedText" format (this is the default) to ensure that the events are distributed with messages and descriptions.
           </p>
           <h4>
-           <a id="_literal_event_logs_event_id_literal">
+           <a id="_event_logs_event_id">
            </a>
            <code class="literal">
             event_logs.event_id
@@ -526,7 +526,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_event_logs_level_literal">
+           <a id="_event_logs_level">
            </a>
            <code class="literal">
             event_logs.level
@@ -629,7 +629,7 @@
     level: critical, error, warning</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_provider_literal">
+           <a id="_event_logs_provider">
            </a>
            <code class="literal">
             event_logs.provider
@@ -673,7 +673,7 @@
 Microsoft-Windows-Eventlog</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_include_xml_literal">
+           <a id="_event_logs_include_xml">
            </a>
            <code class="literal">
             event_logs.include_xml
@@ -702,7 +702,7 @@
     include_xml: true</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_tags_literal">
+           <a id="_event_logs_tags">
            </a>
            <code class="literal">
             event_logs.tags
@@ -754,7 +754,7 @@
       customer_id: 51415432</pre>
           </div>
           <h4>
-           <a id="_literal_event_logs_fields_under_root_literal">
+           <a id="_event_logs_fields_under_root">
            </a>
            <code class="literal">
             event_logs.fields_under_root
@@ -775,7 +775,7 @@
            sub-dictionary. If the custom field names conflict with other field names added by Winlogbeat, then the custom fields overwrite the other fields.
           </p>
           <h4>
-           <a id="_literal_event_logs_processors_literal">
+           <a id="_event_logs_processors">
            </a>
            <code class="literal">
             event_logs.processors
@@ -797,7 +797,7 @@
            for information about specifying processors in your config.
           </p>
           <h4>
-           <a id="_literal_event_logs_no_more_events_literal">
+           <a id="_event_logs_no_more_events">
            </a>
            <code class="literal">
             event_logs.no_more_events
html/en/beats/winlogbeat/master/configure-cloud-id.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configure-cloud-id.html
@@ -197,7 +197,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_cloud_id_literal">
+               <a id="_cloud_id">
                </a>
                <code class="literal">
                 cloud.id
@@ -226,7 +226,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_cloud_auth_literal">
+               <a id="_cloud_auth">
                </a>
                <code class="literal">
                 cloud.auth
html/en/beats/winlogbeat/master/configuring-howto-winlogbeat.html
html/en/beats/winlogbeat/master/configuring-ingest-node.html
html/en/beats/winlogbeat/master/configuring-internal-queue.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/configuring-internal-queue.html
@@ -259,7 +259,7 @@
            config file:
           </p>
           <h5>
-           <a id="_literal_events_literal">
+           <a id="_events">
            </a>
            <code class="literal">
             events
@@ -275,7 +275,7 @@
            The default value is 4096 events.
           </p>
           <h5>
-           <a id="_literal_flush_min_events_literal">
+           <a id="_flush_min_events">
            </a>
            <code class="literal">
             flush.min_events
@@ -291,7 +291,7 @@
            The default value is 2048.
           </p>
           <h5>
-           <a id="_literal_flush_timeout_literal">
+           <a id="_flush_timeout">
            </a>
            <code class="literal">
             flush.timeout
@@ -385,7 +385,7 @@
            config file:
           </p>
           <h5>
-           <a id="_literal_file_path_literal">
+           <a id="_file_path">
            </a>
            <code class="literal">
             file.path
@@ -401,7 +401,7 @@
            The default value is "${path.data}/spool.dat".
           </p>
           <h5>
-           <a id="_literal_file_permissions_literal">
+           <a id="_file_permissions">
            </a>
            <code class="literal">
             file.permissions
@@ -421,7 +421,7 @@
            The default value is 0600.
           </p>
           <h5>
-           <a id="_literal_file_size_literal">
+           <a id="_file_size">
            </a>
            <code class="literal">
             file.size
@@ -457,7 +457,7 @@
            </div>
           </div>
           <h5>
-           <a id="_literal_file_page_size_literal">
+           <a id="_file_page_size">
            </a>
            <code class="literal">
             file.page_size
@@ -504,7 +504,7 @@
            </div>
           </div>
           <h5>
-           <a id="_literal_file_prealloc_literal">
+           <a id="_file_prealloc">
            </a>
            <code class="literal">
             file.prealloc
@@ -551,7 +551,7 @@
            .
           </p>
           <h5>
-           <a id="_literal_write_buffer_size_literal">
+           <a id="_write_buffer_size">
            </a>
            <code class="literal">
             write.buffer_size
@@ -570,7 +570,7 @@
            The default value is 1MiB.
           </p>
           <h5>
-           <a id="_literal_write_codec_literal">
+           <a id="_write_codec">
            </a>
            <code class="literal">
             write.codec
@@ -598,7 +598,7 @@
            .
           </p>
           <h5>
-           <a id="_literal_write_flush_timeout_literal">
+           <a id="_write_flush_timeout">
            </a>
            <code class="literal">
             write.flush.timeout
@@ -622,7 +622,7 @@
            The default value is 1s.
           </p>
           <h5>
-           <a id="_literal_write_flush_events_literal">
+           <a id="_write_flush_events">
            </a>
            <code class="literal">
             write.flush.events
@@ -638,7 +638,7 @@
            The default value is 16384.
           </p>
           <h5>
-           <a id="_literal_read_flush_timeout_literal">
+           <a id="_read_flush_timeout">
            </a>
            <code class="literal">
             read.flush.timeout
html/en/beats/winlogbeat/master/configuring-output.html
html/en/beats/winlogbeat/master/configuring-ssl-logstash.html
html/en/beats/winlogbeat/master/connection-problem.html
html/en/beats/winlogbeat/master/console-output.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/console-output.html
@@ -210,7 +210,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_pretty_literal">
+                <a id="_pretty">
                 </a>
                 <code class="literal">
                  pretty
@@ -235,7 +235,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal_4">
+                <a id="_codec_4">
                 </a>
                 <code class="literal">
                  codec
@@ -271,7 +271,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_6">
+                <a id="_enabled_6">
                 </a>
                 <code class="literal">
                  enabled
@@ -295,7 +295,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_5">
+                <a id="_bulk_max_size_5">
                 </a>
                 <code class="literal">
                  bulk_max_size
html/en/beats/winlogbeat/master/contributing-to-beats.html
html/en/beats/winlogbeat/master/convert.html
html/en/beats/winlogbeat/master/dashboard-fields-incorrect.html
html/en/beats/winlogbeat/master/decode-base64-field.html
html/en/beats/winlogbeat/master/decode-json-fields.html
html/en/beats/winlogbeat/master/decompress-gzip-field.html
html/en/beats/winlogbeat/master/defining-processors.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/defining-processors.html
@@ -306,20 +306,24 @@
               At the top-level in the configuration. The processor is applied to all data collected by Winlogbeat.
              </li>
              <li class="listitem">
-              <p class="simpara">
-               Under a specific event log shipper. The processor is applied to the data collected for that event log shipper. For example:
-              </p>
-              <div class="pre_wrapper lang-yaml">
-               <pre class="programlisting prettyprint lang-yaml">winlogbeat.event_logs:
+              Under a specific event log shipper. The processor is applied to the data collected for that event log shipper.
+             </li>
+            </ul>
+           </div>
+           <p>
+            For example:
+           </p>
+           <p>
+            +
+           </p>
+           <div class="pre_wrapper lang-yaml">
+            <pre class="programlisting prettyprint lang-yaml">winlogbeat.event_logs:
 - name: &lt;network_shipper_name&gt;
   processors:
   - &lt;processor_name&gt;:
       when:
         &lt;condition&gt;
       &lt;parameters&gt;</pre>
-              </div>
-             </li>
-            </ul>
            </div>
           </div>
           <div class="section">
html/en/beats/winlogbeat/master/diff-logstash-beats.html
html/en/beats/winlogbeat/master/directory-layout.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/directory-layout.html
@@ -210,10 +210,6 @@
                </p>
               </td>
               <td align="left" valign="top">
-               <p>
-                <code class="literal">
-                </code>
-               </p>
               </td>
               <td align="left" valign="top">
                <p>
@@ -246,10 +242,6 @@
                </p>
               </td>
               <td align="left" valign="top">
-               <p>
-                <code class="literal">
-                </code>
-               </p>
               </td>
              </tr>
              <tr>
html/en/beats/winlogbeat/master/dissect.html
html/en/beats/winlogbeat/master/drop-event.html
html/en/beats/winlogbeat/master/drop-fields.html
html/en/beats/winlogbeat/master/elasticsearch-output.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/elasticsearch-output.html
@@ -293,7 +293,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal">
+                <a id="_enabled">
                 </a>
                 <code class="literal">
                  enabled
@@ -417,7 +417,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_level_literal">
+                <a id="_compression_level">
                 </a>
                 <code class="literal">
                  compression_level
@@ -444,7 +444,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_escape_html_literal">
+                <a id="_escape_html">
                 </a>
                 <code class="literal">
                  escape_html
@@ -476,7 +476,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal">
+                <a id="_worker">
                 </a>
                 <code class="literal">
                  worker
@@ -500,7 +500,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_username_literal">
+                <a id="_username">
                 </a>
                 <code class="literal">
                  username
@@ -521,7 +521,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_literal">
+                <a id="_password">
                 </a>
                 <code class="literal">
                  password
@@ -542,7 +542,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_parameters_literal">
+                <a id="_parameters">
                 </a>
                 <code class="literal">
                  parameters
@@ -627,7 +627,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_headers_literal">
+                <a id="_headers">
                 </a>
                 <code class="literal">
                  headers
@@ -655,7 +655,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_proxy_url_literal">
+                <a id="_proxy_url">
                 </a>
                 <code class="literal">
                  proxy_url
@@ -1304,7 +1304,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal">
+                <a id="_max_retries">
                 </a>
                 <code class="literal">
                  max_retries
@@ -1329,7 +1329,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal">
+                <a id="_bulk_max_size">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -1367,7 +1367,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_init_literal">
+                <a id="_backoff_init">
                 </a>
                 <code class="literal">
                  backoff.init
@@ -1396,7 +1396,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_max_literal">
+                <a id="_backoff_max">
                 </a>
                 <code class="literal">
                  backoff.max
@@ -1417,7 +1417,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal">
+                <a id="_timeout">
                 </a>
                 <code class="literal">
                  timeout
@@ -1438,7 +1438,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal">
+                <a id="_ssl">
                 </a>
                 <code class="literal">
                  ssl
html/en/beats/winlogbeat/master/enable-winlogbeat-debugging.html
html/en/beats/winlogbeat/master/error-found-unexpected-character.html
html/en/beats/winlogbeat/master/error-loading-config.html
html/en/beats/winlogbeat/master/exported-fields-beat-common.html
html/en/beats/winlogbeat/master/exported-fields-cloud.html
html/en/beats/winlogbeat/master/exported-fields-docker-processor.html
6html/en/beats/winlogbeat/master/exported-fields-ecs.html
html/en/beats/winlogbeat/master/exported-fields-eventlog.html
html/en/beats/winlogbeat/master/exported-fields-host-processor.html
html/en/beats/winlogbeat/master/exported-fields-jolokia-autodiscover.html
html/en/beats/winlogbeat/master/exported-fields-kubernetes-processor.html
html/en/beats/winlogbeat/master/exported-fields-process.html
html/en/beats/winlogbeat/master/exported-fields-security.html
html/en/beats/winlogbeat/master/exported-fields-sysmon.html
html/en/beats/winlogbeat/master/exported-fields-winlog.html
html/en/beats/winlogbeat/master/exported-fields.html
html/en/beats/winlogbeat/master/extract-array.html
html/en/beats/winlogbeat/master/faq.html
html/en/beats/winlogbeat/master/feature-roles.html
html/en/beats/winlogbeat/master/file-output.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/file-output.html
@@ -214,7 +214,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_5">
+                <a id="_enabled_5">
                 </a>
                 <code class="literal">
                  enabled
@@ -259,7 +259,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_filename_literal">
+                <a id="_filename">
                 </a>
                 <code class="literal">
                  filename
@@ -280,7 +280,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_rotate_every_kb_literal">
+                <a id="_rotate_every_kb">
                 </a>
                 <code class="literal">
                  rotate_every_kb
@@ -301,7 +301,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_number_of_files_literal">
+                <a id="_number_of_files">
                 </a>
                 <code class="literal">
                  number_of_files
@@ -328,7 +328,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_permissions_literal">
+                <a id="_permissions">
                 </a>
                 <code class="literal">
                  permissions
@@ -349,7 +349,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal_3">
+                <a id="_codec_3">
                 </a>
                 <code class="literal">
                  codec
html/en/beats/winlogbeat/master/filtering-and-enhancing-data.html
html/en/beats/winlogbeat/master/getting-help.html
html/en/beats/winlogbeat/master/http-endpoint.html
html/en/beats/winlogbeat/master/ilm.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/ilm.html
@@ -283,7 +283,7 @@
           <p>
            The index lifecycle write alias name. The default is
            <code class="literal">
-            winlogbeat-{agent.version}
+            winlogbeat-%{agent.version}
            </code>
            . Setting this option changes the alias name.
           </p>
html/en/beats/winlogbeat/master/include-fields.html
html/en/beats/winlogbeat/master/index.html
html/en/beats/winlogbeat/master/kafka-output.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/kafka-output.html
@@ -264,7 +264,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_3">
+                <a id="_enabled_3">
                 </a>
                 <code class="literal">
                  enabled
@@ -292,7 +292,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_hosts_literal">
+                <a id="_hosts">
                 </a>
                 <code class="literal">
                  hosts
@@ -313,7 +313,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_version_literal">
+                <a id="_version">
                 </a>
                 <code class="literal">
                  version
@@ -355,7 +355,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_username_literal_2">
+                <a id="_username_2">
                 </a>
                 <code class="literal">
                  username
@@ -376,7 +376,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_literal_2">
+                <a id="_password_2">
                 </a>
                 <code class="literal">
                  password
@@ -601,7 +601,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_key_literal">
+                <a id="_key">
                 </a>
                 <code class="literal">
                  key
@@ -622,7 +622,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_partition_literal">
+                <a id="_partition">
                 </a>
                 <code class="literal">
                  partition
@@ -728,7 +728,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_client_id_literal">
+                <a id="_client_id">
                 </a>
                 <code class="literal">
                  client_id
@@ -749,7 +749,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal_3">
+                <a id="_worker_3">
                 </a>
                 <code class="literal">
                  worker
@@ -770,7 +770,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal">
+                <a id="_codec">
                 </a>
                 <code class="literal">
                  codec
@@ -802,7 +802,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_metadata_literal">
+                <a id="_metadata">
                 </a>
                 <code class="literal">
                  metadata
@@ -891,7 +891,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal_3">
+                <a id="_max_retries_3">
                 </a>
                 <code class="literal">
                  max_retries
@@ -916,7 +916,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_3">
+                <a id="_bulk_max_size_3">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -937,7 +937,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_flush_frequency_literal">
+                <a id="_bulk_flush_frequency">
                 </a>
                 <code class="literal">
                  bulk_flush_frequency
@@ -958,7 +958,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal_3">
+                <a id="_timeout_3">
                 </a>
                 <code class="literal">
                  timeout
@@ -979,7 +979,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_broker_timeout_literal">
+                <a id="_broker_timeout">
                 </a>
                 <code class="literal">
                  broker_timeout
@@ -1000,7 +1000,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_channel_buffer_size_literal">
+                <a id="_channel_buffer_size">
                 </a>
                 <code class="literal">
                  channel_buffer_size
@@ -1021,7 +1021,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_keep_alive_literal">
+                <a id="_keep_alive">
                 </a>
                 <code class="literal">
                  keep_alive
@@ -1042,7 +1042,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_literal">
+                <a id="_compression">
                 </a>
                 <code class="literal">
                  compression
@@ -1083,7 +1083,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_level_literal_3">
+                <a id="_compression_level_3">
                 </a>
                 <code class="literal">
                  compression_level
@@ -1135,7 +1135,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_required_acks_literal">
+                <a id="_required_acks">
                 </a>
                 <code class="literal">
                  required_acks
@@ -1159,7 +1159,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal_3">
+                <a id="_ssl_3">
                 </a>
                 <code class="literal">
                  ssl
html/en/beats/winlogbeat/master/keystore.html
html/en/beats/winlogbeat/master/linux-seccomp.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/linux-seccomp.html
@@ -356,6 +356,12 @@
             <dd>
              A list of system call names. The system call name must exist for the runtime architecture, otherwise an error will be logged and the filter will not be installed. At least one system call must be defined.
             </dd>
+           </dl>
+          </div>
+          <div class="variablelist">
+           <a id="seccomp-policy-config-action">
+           </a>
+           <dl class="variablelist">
             <dt>
              <span class="term">
               <span class="strong strong">
@@ -376,8 +382,6 @@
               is executed. This is required. These are the available action values. The actions that are available depend on the kernel version.
              </p>
              <div class="itemizedlist">
-              <a id="seccomp-policy-config-action">
-              </a>
               <ul class="itemizedlist" type="disc">
                <li class="listitem">
                 <code class="literal">
html/en/beats/winlogbeat/master/load-kibana-dashboards.html
html/en/beats/winlogbeat/master/logstash-output.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/logstash-output.html
@@ -180,6 +180,9 @@
             <img alt="Important" src="images/icons/important.png"/>
            </div>
            <div class="admon_content">
+            <h3>
+             Prerequisite
+            </h3>
             <p>
              To send events to Logstash, you also need to create a Logstash configuration pipeline that listens for incoming Beats connections and indexes the received events into Elasticsearch. For more information, see the section about
              <a class="ulink" href="https://www.elastic.co/guide/en/elastic-stack-get-started/master/get-started-elastic-stack.html#logstash-setup" target="_top">
@@ -518,7 +521,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_2">
+                <a id="_enabled_2">
                 </a>
                 <code class="literal">
                  enabled
@@ -566,7 +569,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_level_literal_2">
+                <a id="_compression_level_2">
                 </a>
                 <code class="literal">
                  compression_level
@@ -593,7 +596,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_escape_html_literal_2">
+                <a id="_escape_html_2">
                 </a>
                 <code class="literal">
                  escape_html
@@ -625,7 +628,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal_2">
+                <a id="_worker_2">
                 </a>
                 <code class="literal">
                  worker
@@ -673,7 +676,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ttl_literal">
+                <a id="_ttl">
                 </a>
                 <code class="literal">
                  ttl
@@ -707,7 +710,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_pipelining_literal">
+                <a id="_pipelining">
                 </a>
                 <code class="literal">
                  pipelining
@@ -732,7 +735,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_proxy_url_literal_2">
+                <a id="_proxy_url_2">
                 </a>
                 <code class="literal">
                  proxy_url
@@ -832,7 +835,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal_2">
+                <a id="_ssl_2">
                 </a>
                 <code class="literal">
                  ssl
@@ -863,7 +866,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal_2">
+                <a id="_timeout_2">
                 </a>
                 <code class="literal">
                  timeout
@@ -884,7 +887,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal_2">
+                <a id="_max_retries_2">
                 </a>
                 <code class="literal">
                  max_retries
@@ -909,7 +912,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_2">
+                <a id="_bulk_max_size_2">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -947,7 +950,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_slow_start_literal">
+                <a id="_slow_start">
                 </a>
                 <code class="literal">
                  slow_start
@@ -979,7 +982,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_init_literal_2">
+                <a id="_backoff_init_2">
                 </a>
                 <code class="literal">
                  backoff.init
@@ -1008,7 +1011,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_max_literal_2">
+                <a id="_backoff_max_2">
                 </a>
                 <code class="literal">
                  backoff.max
html/en/beats/winlogbeat/master/metadata-missing.html
html/en/beats/winlogbeat/master/monitoring-internal-collection.html
html/en/beats/winlogbeat/master/monitoring-metricbeat-collection.html
html/en/beats/winlogbeat/master/monitoring-shows-fewer-than-expected-beats.html
html/en/beats/winlogbeat/master/monitoring.html
html/en/beats/winlogbeat/master/processor-dns.html
html/en/beats/winlogbeat/master/processor-registered-domain.html
html/en/beats/winlogbeat/master/processor-script.html
html/en/beats/winlogbeat/master/processor-timestamp.html
html/en/beats/winlogbeat/master/reading-from-evtx.html
html/en/beats/winlogbeat/master/redis-output.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/redis-output.html
@@ -240,7 +240,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_4">
+                <a id="_enabled_4">
                 </a>
                 <code class="literal">
                  enabled
@@ -264,7 +264,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_hosts_literal_2">
+                <a id="_hosts_2">
                 </a>
                 <code class="literal">
                  hosts
@@ -305,7 +305,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_index_literal">
+                <a id="_index">
                 </a>
                 <code class="literal">
                  index
@@ -534,7 +534,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_literal_3">
+                <a id="_password_3">
                 </a>
                 <code class="literal">
                  password
@@ -555,7 +555,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_db_literal">
+                <a id="_db">
                 </a>
                 <code class="literal">
                  db
@@ -576,7 +576,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_datatype_literal">
+                <a id="_datatype">
                 </a>
                 <code class="literal">
                  datatype
@@ -621,7 +621,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal_2">
+                <a id="_codec_2">
                 </a>
                 <code class="literal">
                  codec
@@ -653,7 +653,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_host_topology_literal">
+                <a id="_host_topology">
                 </a>
                 <code class="literal">
                  host_topology
@@ -665,16 +665,18 @@
               </div>
              </div>
             </div>
-            <div class="warning admon">
-             <div class="icon">
-              <img alt="Warning" src="images/icons/warning.png"/>
-             </div>
-             <div class="admon_content">
-              <p>
+            <p>
+             <span class="Admonishment">
+              [
+              <span class="Admonishment-version u-mono u-strikethrough">
+               5.0.0
+              </span>
+              ]
+              <span class="Admonishment-detail">
                Deprecated in 5.0.0.
-              </p>
-             </div>
-            </div>
+              </span>
+             </span>
+            </p>
             <p>
              The Redis host to connect to when using topology map support. Topology map support is disabled if this option is not set.
             </p>
@@ -684,7 +686,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_topology_literal">
+                <a id="_password_topology">
                 </a>
                 <code class="literal">
                  password_topology
@@ -696,16 +698,18 @@
               </div>
              </div>
             </div>
-            <div class="warning admon">
-             <div class="icon">
-              <img alt="Warning" src="images/icons/warning.png"/>
-             </div>
-             <div class="admon_content">
-              <p>
+            <p>
+             <span class="Admonishment">
+              [
+              <span class="Admonishment-version u-mono u-strikethrough">
+               5.0.0
+              </span>
+              ]
+              <span class="Admonishment-detail">
                Deprecated in 5.0.0.
-              </p>
-             </div>
-            </div>
+              </span>
+             </span>
+            </p>
             <p>
              The password to use for authenticating with the Redis topology server. The default is no authentication.
             </p>
@@ -715,7 +719,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_db_topology_literal">
+                <a id="_db_topology">
                 </a>
                 <code class="literal">
                  db_topology
@@ -727,16 +731,18 @@
               </div>
              </div>
             </div>
-            <div class="warning admon">
-             <div class="icon">
-              <img alt="Warning" src="images/icons/warning.png"/>
-             </div>
-             <div class="admon_content">
-              <p>
+            <p>
+             <span class="Admonishment">
+              [
+              <span class="Admonishment-version u-mono u-strikethrough">
+               5.0.0
+              </span>
+              ]
+              <span class="Admonishment-detail">
                Deprecated in 5.0.0.
-              </p>
-             </div>
-            </div>
+              </span>
+             </span>
+            </p>
             <p>
              The Redis database number where the topology information is stored. The default is 1.
             </p>
@@ -746,7 +752,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal_4">
+                <a id="_worker_4">
                 </a>
                 <code class="literal">
                  worker
@@ -771,7 +777,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_loadbalance_literal">
+                <a id="_loadbalance">
                 </a>
                 <code class="literal">
                  loadbalance
@@ -792,7 +798,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal_4">
+                <a id="_timeout_4">
                 </a>
                 <code class="literal">
                  timeout
@@ -813,7 +819,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_init_literal_3">
+                <a id="_backoff_init_3">
                 </a>
                 <code class="literal">
                  backoff.init
@@ -842,7 +848,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_max_literal_3">
+                <a id="_backoff_max_3">
                 </a>
                 <code class="literal">
                  backoff.max
@@ -863,7 +869,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal_4">
+                <a id="_max_retries_4">
                 </a>
                 <code class="literal">
                  max_retries
@@ -888,7 +894,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_4">
+                <a id="_bulk_max_size_4">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -926,7 +932,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal_4">
+                <a id="_ssl_4">
                 </a>
                 <code class="literal">
                  ssl
@@ -957,7 +963,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_proxy_url_literal_3">
+                <a id="_proxy_url_3">
                 </a>
                 <code class="literal">
                  proxy_url
html/en/beats/winlogbeat/master/rename-fields.html
html/en/beats/winlogbeat/master/securing-beats.html
html/en/beats/winlogbeat/master/securing-communication-elasticsearch.html
html/en/beats/winlogbeat/master/securing-winlogbeat.html
html/en/beats/winlogbeat/master/setting-up-and-running.html
html/en/beats/winlogbeat/master/setup-kibana-endpoint.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/setup-kibana-endpoint.html
@@ -206,7 +206,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_setup_kibana_host_literal">
+           <a id="_setup_kibana_host">
            </a>
            <code class="literal">
             setup.kibana.host
@@ -329,7 +329,7 @@
 setup.kibana.path: /kibana</pre>
           </div>
           <h4>
-           <a id="_literal_setup_kibana_username_literal">
+           <a id="_setup_kibana_username">
            </a>
            <code class="literal">
             setup.kibana.username
@@ -346,7 +346,7 @@
            specified for the Elasticsearch output.
           </p>
           <h4>
-           <a id="_literal_setup_kibana_password_literal">
+           <a id="_setup_kibana_password">
            </a>
            <code class="literal">
             setup.kibana.password
@@ -376,7 +376,7 @@
            An HTTP path prefix that is prepended to the HTTP API calls. This is useful for the cases where Kibana listens behind an HTTP reverse proxy that exports the API under a custom prefix.
           </p>
           <h4>
-           <a id="_literal_setup_kibana_ssl_enabled_literal">
+           <a id="_setup_kibana_ssl_enabled">
            </a>
            <code class="literal">
             setup.kibana.ssl.enabled
html/en/beats/winlogbeat/master/shutdown.html
html/en/beats/winlogbeat/master/ssl-client-fails.html
html/en/beats/winlogbeat/master/toc.html
html/en/beats/winlogbeat/master/troubleshooting.html
html/en/beats/winlogbeat/master/upgrading-winlogbeat.html
html/en/beats/winlogbeat/master/using-environ-vars.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/using-environ-vars.html
@@ -472,7 +472,7 @@
              <code class="literal">
               \
              </code>
-             ) will be interpreted as an escape character. :standalone!:
+             ) will be interpreted as an escape character.
             </p>
            </div>
           </div>
html/en/beats/winlogbeat/master/view-kibana-dashboards.html
html/en/beats/winlogbeat/master/winlogbeat-configuration.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/winlogbeat-configuration.html
@@ -438,17 +438,17 @@
                      </a>
                      or equivalent privileges.
                     </p>
-                    <p>
-                     For more information, see
-                     <a class="xref" href="securing-winlogbeat.html" title="Securing Winlogbeat">
-                      Securing Winlogbeat
-                     </a>
-                     .
-                    </p>
                    </td>
                   </tr>
                  </table>
                 </div>
+                <p class="simpara">
+                 For more information, see
+                 <a class="xref" href="securing-winlogbeat.html" title="Securing Winlogbeat">
+                  Securing Winlogbeat
+                 </a>
+                 .
+                </p>
                </li>
               </ul>
              </div>
html/en/beats/winlogbeat/master/winlogbeat-geoip.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/winlogbeat-geoip.html
@@ -231,8 +231,8 @@
               </code>
               processors to add location information to the event. For example, you can use the Console in Kibana to create the following pipeline:
              </p>
-             <div class="pre_wrapper lang-json">
-              <pre class="programlisting prettyprint lang-json">PUT _ingest/pipeline/geoip-info
+             <div class="pre_wrapper lang-console">
+              <pre class="programlisting prettyprint lang-console">PUT _ingest/pipeline/geoip-info
 {
   "description": "Add geoip info",
   "processors": [
html/en/beats/winlogbeat/master/winlogbeat-getting-started.html
html/en/beats/winlogbeat/master/winlogbeat-installation.html
html/en/beats/winlogbeat/master/winlogbeat-module-security.html
html/en/beats/winlogbeat/master/winlogbeat-module-sysmon.html
html/en/beats/winlogbeat/master/winlogbeat-modules.html
--- /dev/fd/63
+++ html/en/beats/winlogbeat/master/winlogbeat-modules.html
@@ -232,6 +232,9 @@
              edit
             </a>
            </h3>
+           <p>
+            pass::[&lt;?edit_url?&gt;]
+           </p>
            <div class="itemizedlist">
             <ul class="itemizedlist" type="disc">
              <li class="listitem">
html/en/beats/winlogbeat/master/winlogbeat-reference-yml.html
html/en/beats/winlogbeat/master/winlogbeat-starting.html
html/en/beats/winlogbeat/master/winlogbeat-template.html
html/en/beats/winlogbeat/master/yaml-tips.html
